zerver/views/streams.py

```diff
@@ -395,10 +391,6 @@ def get_streams_backend(request, user_profile,
                              include_all_active)
     return json_success({"streams": streams})
 
-def get_public_streams_backend(request, user_profile):
-    return get_streams_backend(request, user_profile, include_public=True,
-                               include_subscribed=False, include_all_active=False)
```

Comment:
@timabbott , In this line include_subscribed is default False while in the function get_streams_backend (line : 389) it is default true. Is it ok?